From c968256eabe9547d20a4b1c811ce4e32f39e4d2d Mon Sep 17 00:00:00 2001
From: Dimitrios Poulios <dpoulios85@gmail.com>
Date: Mon, 4 Nov 2024 19:13:08 +0200
Subject: [PATCH] Disable mkeficapsule tool

Disable mkeficapsule tool to overcome riscv64 build error in
alpine 3.16

Signed-off-by: Dimitrios Poulios <dpoulios85@gmail.com>
---
 configs/qemu-riscv64_smode_defconfig | 1 +
 configs/qemu-x86_64_defconfig        | 1 +
 configs/rpi_4_defconfig              | 1 +
 3 files changed, 3 insertions(+)

diff --git a/configs/qemu-riscv64_smode_defconfig b/configs/qemu-riscv64_smode_defconfig
index 6baad1de..bd58f4cc 100644
--- a/configs/qemu-riscv64_smode_defconfig
+++ b/configs/qemu-riscv64_smode_defconfig
@@ -23,3 +23,4 @@ CONFIG_SYS_RELOC_GD_ENV_ADDR=y
 CONFIG_DM_MTD=y
 CONFIG_FLASH_SHOW_PROGRESS=0
 CONFIG_SYS_MAX_FLASH_BANKS=2
+# CONFIG_TOOLS_MKEFICAPSULE is not set
diff --git a/configs/qemu-x86_64_defconfig b/configs/qemu-x86_64_defconfig
index 008eb46a..bd7664f4 100644
--- a/configs/qemu-x86_64_defconfig
+++ b/configs/qemu-x86_64_defconfig
@@ -89,3 +89,4 @@ CONFIG_SPL_VIDEO=y
 # CONFIG_SPL_USE_TINY_PRINTF is not set
 CONFIG_GENERATE_ACPI_TABLE=y
 # CONFIG_GZIP is not set
+# CONFIG_TOOLS_MKEFICAPSULE is not set
diff --git a/configs/rpi_4_defconfig b/configs/rpi_4_defconfig
index db5bbe86..fe44933d 100644
--- a/configs/rpi_4_defconfig
+++ b/configs/rpi_4_defconfig
@@ -71,3 +71,4 @@ CONFIG_VIDEO_BCM2835=y
 CONFIG_CONSOLE_SCROLL_LINES=10
 CONFIG_PHYS_TO_BUS=y
 CONFIG_TPM=y
+# CONFIG_TOOLS_MKEFICAPSULE is not set
-- 
2.45.2

